src/main/java/capstone/mfslbackend/controller/TeamController.java

```diff
@@ -46,4 +46,6 @@ public ResponseEntity<List<Player>> getPlayersOnTeam(@RequestParam Long teamId)
         if (CollectionUtils.isEmpty(players)) return ResponseEntity.notFound().build();
         return ResponseEntity.ok(players);
     }
+
```

Comment:
A useful addition could be get all games for team that calls the function getGamesForTeam which you already wrote